content/en/blog/_posts/2024-12-11-volume-group-snapshot-beta.md

```diff
@@ -0,0 +1,373 @@
+---
+layout: blog
+title: "Kubernetes 1.32: Moving Volume Group Snapshots to Beta"
+date: 2024-12-11
+slug: kubernetes-1-32-volume-group-snapshot-beta
+draft: true
+author: >
+   Xing Yang (VMware by Broadcom)
+---
+
+Volume group snapshot was introduced as an
+[Alpha feature in Kubernetes v1.27](https://kubernetes.io/blog/2023/05/08/kubernetes-1-27-volume-group-snapshot-alpha/).
```

Comment:
Try this:
```suggestion
Volume group snapshots were [introduced](/blog/2023/05/08/kubernetes-1-27-volume-group-snapshot-alpha/)
as an Alpha feature with the Kubernetes 1.27 release.
```
(I'm also avoiding a sentence-end period after a version number, here)

Reply:
Done

Reply:
You should fix the hyperlink.